lib/src/download_utils.dart

```diff
@@ -166,7 +167,7 @@ class UrlChecker {
     return false;
   }
 
-  Future<UrlStatus> checkStatus(String url) async {
+  Future<UrlStatus> checkStatus(String url, {String packageName}) async {
```

Comment:
Alternatively this pattern logic does not need to exist in pana, but could accept a bool 'isInternal' that could be provided by the pub site code.

Reply:
Yeah, it could become part of `InspectOptions`.